release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -183,6 +183,13 @@ The IPsec tunnel is not used for network traffic between pods that are configure
 
 The SR-IOV Network Operator is enhanced to support an additional field, `spec.nicSelector.netFilter`, in the custom resource for an SR-IOV network node policy. You can use the new field to specify an {rh-openstack} network by the network ID. For more information, see xref:../networking/hardware_networks/configuring-sriov-device.adoc[Configuring an SR-IOV network device].
 
+[id="ocp-4-7-preparation-HAProxy-2.2"]
+==== Adjusting HTTP header names
+
+If legacy applications are sensitive to the capitalization of HTTP header names, use the Ingress Controller `spec.httpHeaders.headerNameCaseAdjustments` API field for a solution to accommodate legacy applications until they can be fixed.
+
+{product-title} is going to update to HAProxy 2.2, which down-cases HTTP header names by default, for example, changing `Host: xyz.com` to `host: xyz.com`. Make sure to add the necessary configuration using `spec.httpHeaders.headerNameCaseAdjustments` before upgrading {product-title} when HAproxy 2.2 is available.
```

Comment:
```suggestion
{product-title} will update to HAProxy 2.2, which down-cases HTTP header names by default, for example, changing `Host: xyz.com` to `host: xyz.com`. Make sure to add the necessary configuration by using `spec.httpHeaders.headerNameCaseAdjustments` before upgrading {product-title} when HAproxy 2.2 is available.
```
Will users know where to set that configuration? If it's not obvious, it might be worth clarifying either here or on the previous line.